client/src/connection/itc/index.ts

```diff
@@ -319,7 +319,7 @@ export class ITCSession extends Session {
           line,
           this._html5FileName,
         );
-        this._onLogFn?.([{ type: "normal", line }]);
+        this._onExecutionLogFn?.([{ type: "normal", line }]);
```

Comment:
shall we distinguish session log here? perhaps
```
if (this._workDirectory) {
    this._onExecutionLogFn?.([{ type: "normal", line }]);
} else {
    this._onSessionLogFn?.([{ type: "normal", line }]);
}

Reply:
fixed in https://github.com/sassoftware/vscode-sas-extension/pull/713/commits/6d6d1e0dd3d0938d427f339603b093b34cf7d2f0